Source/EasyNetQ/ConnectionConfiguration.cs

```diff
@@ -90,21 +92,29 @@ private void SetDefaultClientProperties(IDictionary<string, object> clientProper
             var hostname = Environment.MachineName;
             var product = Product ?? applicationName;
             var platform = Platform ?? hostname;
+            var name = Name ?? applicationName;
```

Comment:
Good point, I don't mind it to be the default, but maybe for backwards compatibility it should be left empty and not be part of the connection string when not specified?

@mantasaudickas  what do you think?